api/Cargo.toml

```diff
@@ -15,6 +15,7 @@ path = "src/main.rs"
 async-trait = "0.1"
 base64 = "0.21"
 boa_engine = "0.17"
+bytesize = {version = "1.3.0", features = ["serde"]}
```

Comment:
All versions here don't specify the patch version. Please, update accordingly.

Reply:
Added the new patched dependencies.